dhall/ghc-src/Dhall/Import/HTTP.hs

```diff
@@ -169,7 +171,7 @@ renderPrettyHttpException url e = case e of
         <>  show e' <> "\n"
 #endif
 
-newManager :: IO Manager
+newManager :: StateT Status IO Manager
 newManager = do
     let settings = HTTP.tlsManagerSettings
 #ifdef MIN_VERSION_http_client
```

Comment:
I know it's unrelated to your patch, but do you think we could remove this outer CPP conditional?